tests/test_image_driver.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python 
+''' VIC Image Driver testing '''
+
+import xarray as xr
+import numpy as np 
+import numpy.testing as npt 
+
+
+# In[28]:
+
+file = '/Users/diana/workdir/VIC_tests_20160331/examples/Example-Image-Stehekin-base-case/results/Stehekin.history.nc'
+domainfile = '/Users/diana/Dropbox/UW/VIC_sample_data/image/Stehekin/parameters/domain.stehekin.20151028.nc'
+
+
+# In[29]:
+
+ds = xr.open_dataset(file)
+ds_domain = xr.open_dataset(domainfile)
```

Comment:
remove these lines as we don't want to run them every time we import this module.